js-miniapp-bridge/src/types/token-data.ts

```diff
@@ -2,15 +2,35 @@
 export interface NativeTokenData {
   token: string;
   validUntil: number;
+  scopes?: NativeTokenPermission;
+}
+
+/** @internal */
+export interface NativeTokenPermission {
+  audience?: string;
+  scopes?: string[];
```

Comment:
I think these properties shouldn't be optional. The iOS/Android side should always return a value.

Reply:
done